Upstream: Should be
Reason: Fixes compilation on musl ppc64le

diff --git a/direct.c b/direct.c
index 67bed20..1112f31 100644
--- a/direct.c
+++ b/direct.c
@@ -28,6 +28,7 @@
 #include <unistd.h>
 #include <fcntl.h>
 #include <errno.h>
+#include <limits.h>
 
 #include <sys/xattr.h>
 
